src/Tribe/App_Shop.php

```diff
@@ -77,11 +79,28 @@ public function add_toolbar_item() {
 		}
 
 		/**
+		 * Registers the plugin assets
+		 */
+		protected function register_assets() {
+			tribe_assets(
+				Tribe__Main::instance(),
+				array(
+					array( 'app-shop-css', 'app-shop.css' ),
+					array( 'app-shop-js', 'app-shop.js' ),
+				),
+				'admin_print_styles-' . $this->admin_page
+			);
+		}
+
+		/**
 		 * Enqueue the styles and script
 		 */
 		public function enqueue() {
-			wp_enqueue_style( 'app-shop', tribe_resource_url( 'app-shop.css', false, null, Tribe__Main::instance() ), array(), apply_filters( 'tribe_events_css_version', Tribe__Main::VERSION ) );
-			wp_enqueue_script( 'app-shop', tribe_resource_url( 'app-shop.js', false, null, Tribe__Main::instance() ), array(), apply_filters( 'tribe_events_js_version', Tribe__Main::VERSION ) );
+
+
+
+			wp_enqueue_style( 'app-shop-css' );
```

Comment:
Yessir! Here's the docblock for [`tribe_asset()`](https://github.com/moderntribe/tribe-common/blob/master/src/functions/template-tags/general.php#L518). The 5th parameter lets you specify an action.